js/components/directionControlVr.js

```diff
@@ -0,0 +1,42 @@
+var React = require('react');
+var classnames = require('classnames');
+
+var DirectionControlVr = React.createClass({
+  getInitialState: function() {
+    return {
+      isTouched: false,
+    };
+  },
+
+  handleEvent: function (ev) {
+    var rotate = ev.type == 'mousedown' || ev.type == 'touchstart';
+    this.props.handleDirection(rotate, this.props.dir);
+    
+    this.setState({
+      isTouched: rotate
+    });
+  },
+
+  render: function () {
+    var baseDirectionClass = 'oo-vr-icon--move';
+    var directionClass = baseDirectionClass + '--' + this.props.dir;
+    var touchedDirectionClass = this.state.isTouched && directionClass + '--touched';
```

Comment:
let's avoid using an operation where you do BOOLEAN && STRING.

Reply:
\+